app/component/enter-room/EnterRoomBackground.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import {Image, ImageBackground, StyleSheet} from 'react-native'
+export default (props) => {
+    return (
+        <ImageBackground
+            style={[styles.backgroundImage, props.style]}
+            source={require('../../assets/enterscreen-background/enterscreen-background.png')}
```

Comment:
does this need to use require() ? To me import would make it look cleaner.

Reply:
Using import was complaining in linter. Is there way to fix it in typescript?